packages/bff/src/providers/clientProvider.ts

```diff
@@ -7,9 +7,9 @@ import {
 } from "pagopa-interop-api-clients";
 import { config } from "../config/config.js";
 
-export type TenantProcessClient = {
-  tenant: ReturnType<typeof tenantApi.createTenantApiClient>;
-};
+export type TenantProcessClient = ReturnType<
+  typeof tenantApi.createTenantApiClient
+>;
```

Comment:
This approach conflicts with this other adopted by @sandrotaje:  https://github.com/pagopa/interop-be-monorepo/pull/663/files#diff-4053f4f2ef3d180bfac42a73b669a5b4497a162c94b4465c15652d422b91b8adR43

Since the tenantApi results in generating many clients due to the different tags, I would go with the approach adopted by Sandro and return an object with all the tenant-process API clients

Reply:
I'm agree with you, done with [this commit](https://github.com/pagopa/interop-be-monorepo/pull/712/commits/43848ef64033f18c7d4a5be8b65b7fbf3e24aaae)